### DISCUSSION TITLE
Any possible to turn a dlna speaker to AirPlay2?

### DISCUSSION THREAD

**Discussion by goodbai1988** [mikebrady/shairport-sync#1713] [Q&A]
I'm new here and already done some research (maybe not enough...). 
Is there any way to connect dlna speaker and recognize it as an output so we can ShairPort-Sync it?

**Comment by mikebrady** [comment]
Thanks for the question. I am not aware of anything like that, I’m afraid. And I don’t know too much about DNLA. Specifically, I don’t know if it can provide any information or control over latency. Without that kind of information, TBH, it is hard to see it as a worthwhile endeavour.

**Reply by goodbai1988** [reply]
Got it! 
Still a BIG THANK for your awesome work!  🥳🥳🥳

**Reply by eleven520** [reply]
I want to use this project to make all the smart speakers in my LAN recognized by homekit and added to HOME and support airplay2 audio output, which is not possible at this stage, right?  Also Big Thanks for your work！！！

**Reply by mikebrady** [reply]
Thanks for the kind words. Unfortunately we don't really know how HomeKit and AirPlay 2 interact, so we are not really making progress in that area...
